core/src/main/scala/akka/persistence/cassandra/healthcheck/AkkaPersistenceCassandraHealthCheck.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra.healthcheck
+
+import akka.actor.ActorSystem
+import akka.event.Logging
+import akka.pattern.{ ask, AskTimeoutException }
+import akka.persistence.Persistence
+import akka.persistence.cassandra.journal.CassandraJournal.{ HealthCheckQuery, HealthCheckResponse }
+import akka.util.Timeout
+
+import scala.concurrent.{ ExecutionContextExecutor, Future }
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class AkkaPersistenceCassandraHealthCheck(system: ActorSystem) extends (() => Future[Boolean]) {
+
+  private[akka] val log = Logging.getLogger(system, getClass)
+  private implicit val ec: ExecutionContextExecutor = system.dispatcher
+  private implicit val timeout: Timeout = 1 second
+
+  override def apply(): Future[Boolean] = {
+    val journalRef = Persistence(system).journalFor("akka.persistence.cassandra.journal")
```

Comment:
the plugin id should also be configurable, for some other things we have named that `plugin-location` and it's one level up, i.e. adding ".journal" here.

```
akka.persistence.cassandra.healthcheck {
  plugin-location = "akka.persistence.cassandra"
}
```

Reply:
Done.